src/battle/overlay_12_0224E4FC.c

```diff
@@ -5541,3 +5542,939 @@ BOOL TryFling(BattleSystem *bsys, BATTLECONTEXT *ctx, int battlerId) {
     
     return TRUE;
 }
+
+void ov12_022565E0(BattleSystem *bsys, BATTLECONTEXT *ctx) {
+    if (GetBattlerHeldItemEffect(ctx, ctx->battlerIdAttacker) == HOLD_EFFECT_BOOST_REPEATED) {
+        if (!(ctx->battleMons[ctx->battlerIdAttacker].status2 & STATUS2_RAGE) &&
+            !(ctx->battleMons[ctx->battlerIdAttacker].status2 & STATUS2_UPROAR) &&
+            !(ctx->linkStatus & (1 << 9)) &&
+            !(ctx->battleMons[ctx->battlerIdAttacker].status2 & STATUS2_LOCKED_INTO_MOVE)) {
+            if (ctx->moveNoMetronome[ctx->battlerIdAttacker] == ctx->moveNoTemp) {
+                if (ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns < 10) {
+                    ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns++;
+                }
+            } else {
+                ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns = 0;
+                ctx->moveNoMetronome[ctx->battlerIdAttacker] = ctx->moveNoTemp;
+            }
+        }
+    } else {
+        ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns = 0;
+    }
+}
+
+void ov12_02256694(BattleSystem *bsys, BATTLECONTEXT *ctx) {
+    if (GetBattlerHeldItemEffect(ctx, ctx->battlerIdAttacker) == HOLD_EFFECT_BOOST_REPEATED) {
+        if ((ctx->moveStatusFlag & MOVE_STATUS_FAIL) &&
+            ctx->moveNoMetronome[ctx->battlerIdAttacker] == ctx->moveNoTemp &&
+            ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns &&
+            !(ctx->selfTurnData[ctx->battlerIdAttacker].rolloutCount) &&
+            !(ctx->battleMons[ctx->battlerIdAttacker].status2 & STATUS2_RAGE) &&
+            !(ctx->battleMons[ctx->battlerIdAttacker].status2 & STATUS2_UPROAR) &&
+            !(ctx->linkStatus & (1 << 9)) &&
+            !(ctx->battleMons[ctx->battlerIdAttacker].status2 & STATUS2_LOCKED_INTO_MOVE)) {
+            ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns--;
+        }
+    } else {
+        ctx->battleMons[ctx->battlerIdAttacker].unk88.metronomeTurns = 0;
+    }
+}
+
+//Related to send out Pokemon crys..?
+int ov12_02256748(BATTLECONTEXT *ctx, int battlerId, int battlerType, BOOL encounter) {
+    int ret;
+    int color;
+    BOOL half;
+    
+    if (encounter == TRUE && (battlerType == 2 || battlerType == 3)) {
```

Comment:
also do we have battler type constants

Reply:
tbh I dont even know what battler type means other than it's most accurate description I can give